### PR TITLE
2020update

### PR DESCRIPTION
**Pull request by Couragyn**
Updated results for 2020. Show them. Fixed 2 decimal point percent bug

### CODE REVIEW

File: db/scripts/import_election_results_2020.rb [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+      until sheet.cell(curr_row, 1) == "Final voting results - Complete"
+        curr_row += 1
+      end
+      dis_last_candidate_row = curr_row - 1
+      dis_first_candidate_row = dis_last_candidate_row - dis_start_row - 1
+      i = 0
+      for curr_row in (dis_start_row + 1)..dis_last_candidate_row
+        party = sheet.cell(curr_row, sheet.first_column + 2)
+        party = 'N/A' if party.nil?
+        candidates[i] = {name: sheet.cell(curr_row, sheet.first_column + 1), party: party, votes_total: sheet.cell(curr_row, sheet.first_column + 3), votes_percent: ((sheet.cell(curr_row, sheet.first_column + 4)).chomp('%')).to_f}
```
Comment (nanstey) [inline db/scripts/import_election_results_2020.rb:40]: breaking this object onto several lines would read better

File: spec/support/create_party_and_election.rb [hunk 2]
```diff
@@ -24,15 +24,18 @@ def party_and_election
...
     {name: 'The Sex Party', abbr: 'SEX'},
     {name: 'The Vancouver Island Party', abbr: 'VIP'},
     {name: 'Unparty: The Consensus-Building Party', abbr: 'UCBP'},
     {name: 'Western Canada Concept Party', abbr: 'WCC'},
+    {name: 'Wexit BC', abbr: 'WX'},
     {name: 'Work Less Party', abbr: 'WLP'},
     {name: 'Your Political Party of BC (YPP)', abbr: 'YPBC'}
   ])
   Election.create!([
+    {year: 2020},
```
Comment (nanstey) [inline spec/support/create_party_and_election.rb:38]: why do we need both this file and `db/scripts/seed_parties.rb` ?

**Review by nanstey — COMMENTED** [review]
Gave it a once over, I assume it works hehe

Just a couple of minor comments :+1:

File: spec/support/create_party_and_election.rb (hunk 2 above)
Comment (Couragyn) [reply spec/support/create_party_and_election.rb:38]: create_party_and_election.rb is in Spec, so it's used for tests

seed_parties.rb is for filling the db with our data

Looks like both are needed

File: db/scripts/import_election_results_2020.rb (hunk 1 above)
Comment (Couragyn) [reply db/scripts/import_election_results_2020.rb]: It's so pretty now

**Pull request merged**